x-pack/plugins/transform/public/app/sections/create_transform/components/aggregation_list/popover_form.tsx

```diff
@@ -48,10 +72,50 @@ export const PopoverForm: React.FC<Props> = ({ defaultData, otherAggNames, onCha
   const [field, setField] = useState(
     isPivotAggsConfigWithUiSupport(defaultData) ? defaultData.field : ''
   );
+  const [percents, setPercents] = useState(getDefaultPercents(defaultData));
 
   const availableFields: SelectOption[] = [];
   const availableAggs: SelectOption[] = [];
 
+  function updateAgg(aggVal: PIVOT_SUPPORTED_AGGS) {
+    setAgg(aggVal);
+    if (aggVal === PIVOT_SUPPORTED_AGGS.PERCENTILES && percents === undefined) {
+      setPercents(PERCENTILES_AGG_DEFAULT_PERCENTS);
+    }
+  }
+
+  function updatePercents(inputValue: string) {
+    const isInputValid = isPercentsInputValid(inputValue);
+    if (isInputValid === true) {
+      setPercents(inputValue.split(',').map(Number));
```

Comment:
maybe `isPercentsInputValid` function could be replaced with `parsePercentsInput` to receive an array of percent values and  avoid doing split and map again.